error: An error occurred: Out of range value for column 'phonenumber' at row 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_WARN_DATA_OUT_OF_RANGE","errno":1264,"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$834J1sLW3e/QytwvdZISXe6iXSAv0MUfbs95QSIMlwuIuIB.Iu/HG","2023-07-13 11:26:27","2023-07-13 11:26:27"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Out of range value for column 'phonenumber' at row 1","sqlState":"22003"},"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$834J1sLW3e/QytwvdZISXe6iXSAv0MUfbs95QSIMlwuIuIB.Iu/HG","2023-07-13 11:26:27","2023-07-13 11:26:27"],"parent":{"code":"ER_WARN_DATA_OUT_OF_RANGE","errno":1264,"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$834J1sLW3e/QytwvdZISXe6iXSAv0MUfbs95QSIMlwuIuIB.Iu/HG","2023-07-13 11:26:27","2023-07-13 11:26:27"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Out of range value for column 'phonenumber' at row 1","sqlState":"22003"},"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.insert (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2490:35)\n    at async signup.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.signupdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/signup.js:20:18)"}
error: An error occurred: Out of range value for column 'phonenumber' at row 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_WARN_DATA_OUT_OF_RANGE","errno":1264,"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$fnfEPfuUd8Sg0cS1HJYqKu83G.0bzHWAOt8nOY4PqGp7empL49nX6","2023-07-13 11:31:37","2023-07-13 11:31:37"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Out of range value for column 'phonenumber' at row 1","sqlState":"22003"},"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$fnfEPfuUd8Sg0cS1HJYqKu83G.0bzHWAOt8nOY4PqGp7empL49nX6","2023-07-13 11:31:37","2023-07-13 11:31:37"],"parent":{"code":"ER_WARN_DATA_OUT_OF_RANGE","errno":1264,"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$fnfEPfuUd8Sg0cS1HJYqKu83G.0bzHWAOt8nOY4PqGp7empL49nX6","2023-07-13 11:31:37","2023-07-13 11:31:37"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Out of range value for column 'phonenumber' at row 1","sqlState":"22003"},"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at async MySQLQueryInterface.insert (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2490:35)\n    at async signup.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.signupdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/signup.js:20:18)"}
error: An error occurred: notNull Violation: messages.message cannot be null,
notNull Violation: messages.userId cannot be null {"errors":[{"instance":{"createdAt":"2023-07-14T03:59:21.022Z","id":null,"updatedAt":"2023-07-14T03:59:21.022Z"},"message":"messages.message cannot be null","origin":"CORE","path":"message","type":"notNull Violation","validatorArgs":[],"validatorKey":"is_null","validatorName":null,"value":null},{"instance":{"createdAt":"2023-07-14T03:59:21.022Z","id":null,"updatedAt":"2023-07-14T03:59:21.022Z"},"message":"messages.userId cannot be null","origin":"CORE","path":"userId","type":"notNull Violation","validatorArgs":[],"validatorKey":"is_null","validatorName":null,"value":null}],"name":"SequelizeValidationError","stack":"SequelizeValidationError: notNull Violation: messages.message cannot be null,\nnotNull Violation: messages.userId cannot be null\n    at InstanceValidator._validate (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:50:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async InstanceValidator._validateAndRunHooks (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:60:7)\n    at async InstanceValidator.validate (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:54:12)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2426:7)\n    at async messages.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.chatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:11:18)"}
error: An error occurred: notNull Violation: groups.group_description cannot be null {"errors":[{"instance":{"createdAt":"2023-07-14T13:05:49.389Z","creator_id":4,"group_id":null,"group_name":"sharpers group","updatedAt":"2023-07-14T13:05:49.389Z"},"message":"groups.group_description cannot be null","origin":"CORE","path":"group_description","type":"notNull Violation","validatorArgs":[],"validatorKey":"is_null","validatorName":null,"value":null}],"name":"SequelizeValidationError","stack":"SequelizeValidationError: notNull Violation: groups.group_description cannot be null\n    at InstanceValidator._validate (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:50:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async InstanceValidator._validateAndRunHooks (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:60:7)\n    at async InstanceValidator.validate (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:54:12)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2426:7)\n    at async groups.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.creategroup (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:10:18)"}
error: An error occurred: Table 'chat_connect1.groups' doesn't exist {"name":"SequelizeDatabaseError","original":{"code":"ER_NO_SUCH_TABLE","errno":1146,"parameters":["sharpener group",4,"2023-07-14 13:10:24","2023-07-14 13:10:24"],"sql":"INSERT INTO `groups` (`group_id`,`group_name`,`creator_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?);","sqlMessage":"Table 'chat_connect1.groups' doesn't exist","sqlState":"42S02"},"parameters":["sharpener group",4,"2023-07-14 13:10:24","2023-07-14 13:10:24"],"parent":{"code":"ER_NO_SUCH_TABLE","errno":1146,"parameters":["sharpener group",4,"2023-07-14 13:10:24","2023-07-14 13:10:24"],"sql":"INSERT INTO `groups` (`group_id`,`group_name`,`creator_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?);","sqlMessage":"Table 'chat_connect1.groups' doesn't exist","sqlState":"42S02"},"sql":"INSERT INTO `groups` (`group_id`,`group_name`,`creator_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?);","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.insert (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2490:35)\n    at async groups.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.creategroup (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:10:18)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Cannot read properties of null (reading 'id') {"stack":"TypeError: Cannot read properties of null (reading 'id')\n    at exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:24:29)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Cannot read properties of null (reading 'id') {"stack":"TypeError: Cannot read properties of null (reading 'id')\n    at exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:24:29)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Cannot read properties of null (reading 'id') {"stack":"TypeError: Cannot read properties of null (reading 'id')\n    at exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:24:29)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1","sqlState":"42000"},"parameters":{},"parent":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1","sqlState":"42000"},"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async signup.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getusersearchdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/usersearch.js:11:18)"}
error: An error occurred: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1","sqlState":"42000"},"parameters":{},"parent":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1","sqlState":"42000"},"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async signup.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getusersearchdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/usersearch.js:11:18)"}
error: An error occurred: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1","sqlState":"42000"},"parameters":{},"parent":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1","sqlState":"42000"},"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async signup.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getusersearchdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/usersearch.js:11:18)"}
error: An error occurred: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1","sqlState":"42000"},"parameters":{},"parent":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1","sqlState":"42000"},"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async signup.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getusersearchdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/usersearch.js:11:18)"}
error: An error occurred: Unknown column 'signup.collate' in 'where clause' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` LIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"Unknown column 'signup.collate' in 'where clause'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` LIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"Unknown column 'signup.collate' in 'where clause'","sqlState":"42S22"},"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` LIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async signup.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getusersearchdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/usersearch.js:11:18)"}
error: An error occurred: data is not defined {"stack":"ReferenceError: data is not defined\n    at exports.getuserdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/user.js:10:5)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: userDetails is not defined {"stack":"ReferenceError: userDetails is not defined\n    at exports.getuserdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/user.js:13:43)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: connect ETIMEDOUT {"name":"SequelizeConnectionError","original":{"code":"ETIMEDOUT","errorno":"ETIMEDOUT","fatal":true,"syscall":"connect"},"parent":{"code":"ETIMEDOUT","errorno":"ETIMEDOUT","fatal":true,"syscall":"connect"},"stack":"SequelizeConnectionError: connect ETIMEDOUT\n    at ConnectionManager.connect (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:102:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ConnectionManager._connect (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:222:24)"}
error: An error occurred: GroupModel is not defined {"stack":"ReferenceError: GroupModel is not defined\n    at exports.getgroupnames (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:25:20)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Cannot read properties of undefined (reading 'id') {"stack":"TypeError: Cannot read properties of undefined (reading 'id')\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:28:28\n    at Array.map (<anonymous>)\n    at exports.getgroupnames (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:26:31)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
